wallets.json

```diff
@@ -29,6 +31,7 @@
             "rpc": "https://ctz.solidwallet.io/api/v3/icon_dex",
             "type": "icon",
             "decimals": 18,
+            "threshold": "0.1",
```

Comment:
ICX gets used easily, especially for BTP, so lets have a larger threshold.

200 ICX should be alright

Reply:
Sure, it was just an example for quick test